{"name": "Передали в поле Login существующий в системе. Ожидаем ошибку 409.", "status": "passed", "description": "Класс тестирования курьера. Создание.", "steps": [{"name": "Логин существует в системе, пароль и имя сгенерировали.", "status": "passed", "start": 1727982863695, "stop": 1727982863890}], "start": 1727982863695, "stop": 1727982863890, "uuid": "8755053d-0313-4243-b8f3-d4d062655200", "historyId": "4e7693f58d2a887047e6e2bd79f2a05e", "testCaseId": "4e7693f58d2a887047e6e2bd79f2a05e", "fullName": "test.test_create_courier.TestCreateCourier#test_cant_create_two_same_logins_true", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "Darkness"}, {"name": "thread", "value": "28392-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_create_courier"}]}